pkg/unifiedpush/client.go

```diff
@@ -31,6 +31,15 @@ type AndroidVariant struct {
 	variant
 }
 
+// webPushVariant is an internal struct used for convenient JSON
+// unmarshalling of the response received from UPS
+type WebPushVariant struct {
+	PrivateKey string
+	PublicKey  string
+	Alias      string
+	variant
```

Comment:
This looks odd to me but I'm not sure why.

Reply:
@grdryn explained this one to me. Apparently it's a normal thing in go. Resolving this comment as nothing needs to be done.